tests/platforms/iOS/xcode/test_mixin.py

```diff
@@ -37,3 +40,39 @@ def test_distribution_path(create_command, first_app_config, tmp_path):
         match=r"WARNING: No distributable artefact has been generated",
     ):
         create_command.distribution_path(first_app_config)
+
+
+def test_verify(create_command, monkeypatch):
+    """If you're on macOS, you can verify tools."""
+    create_command.tools.host_os = "Darwin"
```

Comment:
What coverage is this adding? Is it purely to evaluate "verify macOS works" on Windows/Linux?

Reply:
I wouldn't consider this one to be adding coverage per se; although, it does add coverage on Windows and Linux.

In my mind, this adds "proper" testing and coverage for iOS verification since it explicitly ensures the expected tools are verified. I think that all of the platforms with tools should have this test.

Furthermore, I've added all of these tests in #1093.

Reply:
I definitely agree that the rest of the test is a much more comprehensive test of the core iOS verification. I guess my question is whether we gain anything by running this test on Windows/Linux. However, it's only verifying that the right workflow is executed (i.e., the right verify methods are invoked), so I guess it doesn't hurt.